pennylane/devices/default_qutrit_mixed.py

```diff
@@ -154,6 +220,27 @@ def f(x):
     >>> jax.grad(f)(jax.numpy.array(1.2))
     DeviceArray(-0.93203914, dtype=float32, weak_type=True)
 
+    .. details::
+        :title: Readout Error
+
+        ``DefaultQutritMixed`` includes readout error support. There are two readout error inputs
+        which apply an error channel to the state after it has
+        been diagonalized for the measurement:
+
+        * ``readout_relaxation_probs``: Defines the readout error inputs of :class:`qml.QutritAmplitudeDamping`
+            channel. This error models state relaxation error that occur during readout of transmon
+            based qutrits. The motivation for this readout error is decribed in
+            [`1 <https://arxiv.org/abs/2003.03307>`_] (Sec II.A).
+        * ``readout_misclassification_probs``: Defines the inputs of :class:`qml.TritFlip` channel.
+            This error models misclassification events in readout. An example of this readout error
+            can be seen in [`2 <https://arxiv.org/abs/2309.11303>`_] (Fig 1a).
```

Comment:
This is not showing up quite right in the docs
![image](https://github.com/PennyLaneAI/pennylane/assets/49409390/58aca382-bbb7-4489-8465-cb12e17985bc)
The subsequent lines for each bullet are overindented.

Reply:
Oh thank you, I couldn't get the indentation to work without just getting rid of new lines, is that an ok solution?